Windows CLI builds very flaky

## Expected Behavior

The GitHub action `CLI Test Windows` fails when it should

## Current Behavior

It frequently fails with the following error, even for trivial, e.g. README changes:

```
Run yarn install --frozen-lockfile
yarn install v1.22.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
error An unexpected error occurred: "https://registry.npmjs.org/@material-ui/icons/-/icons-4.9.1.tgz: ESOCKETTIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "D:\\a\\backstage\\backstage\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
##[error]Process completed with exit code 1.
```
## Possible Solution

???

## Steps to Reproduce

Trigger a CLI build by making a change to the CLI package

## Context

https://github.com/spotify/backstage/runs/798389403?check_suite_focus=true
https://github.com/spotify/backstage/runs/798498798?check_suite_focus=true

--- a/.yarnrc
+++ b/.yarnrc
@@ -5,3 +5,4 @@
 registry "https://registry.npmjs.org/"
 lastUpdateCheck 1580389148099
 yarn-path ".yarn/releases/yarn-1.22.1.js"
+network-timeout 600000

